src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -598,6 +598,16 @@ export class ConnectionWidget extends lifecycle.Disposable {
 			this._userNameInputBox.enable();
 			this._passwordInputBox.enable();
 			this._rememberPasswordCheckBox.enabled = true;
+
+			const recentConnections = this._connectionManagementService.getRecentConnections();
```

Comment:
Is recent connections the right one to use here? Recent connections are temporary and can be cleared at any time, shouldn't we be using saved connections instead? I suppose using both is an option too in case they didn't save the connection when previously connected...